WcaOnRails/app/views/registrations/do_actions_for_selected.js.erb

```diff
@@ -44,7 +44,9 @@ $registrationsTable.bootstrapTable('uncheckAll');
 $registrationsTable.find('tr.selected').removeClass('selected');
 
 // Update registration table footers with summary info
+<% registrations = @competition.registrations.includes(:user)
+   registrations = registrations.includes(:events) if @show_events %>
 <% [:pending, :accepted, :deleted].each do |status| %>
-  <% registrations = @competition.registrations.includes(:events, :user).send(status) %>
+  <% registrations = registrations.public_send(status) %>
```

Comment:
Hmm... this looks concerning. Is this overwriting the same `registrations` variable each time through the loop? If so, aren't we going to end up looking at `registrations.pending` followed by `registrations.pending.accepted`, followed by `registrations.pending.accepted.deleted`? I expect that neither of the last 2 will show anything.